docker-compose.yml

```diff
@@ -41,3 +41,19 @@ services:
         - "9300:9300"
       environment:
         - discovery.type=single-node
+
+    musl-builder:
```

Comment:
was this just used for testing? or is it something we will need in the long run?

Reply:
It is just for testing the build.